php-classes/Emergence/Connectors/Exceptions/SyncException.php

```diff
@@ -29,18 +29,13 @@ public function getContext($key = null)
         return $this->context;
     }
 
-    protected static function interpolate($message, array $context = array())
+    public static function interpolate($message, array $context = [])
```

Comment:
@nbey why did you need to switch this to public? We always want to err on the side of protected as it leaves more surface area of the class safe for reworking later without risking breaking its consumers. Also, since this matches a function defined elsewhere, we should consider a common place to put it

Rather than having a static method within the class that a consumer needs to call and pass in two other bits from a class instance (picture all the zig zags in a call diagram of that), a better strategy would be to hide all these details within the class and expose one public instance method `getInterpolatedMessage()`